packages/models-library/src/models_library/api_schemas_datcore_adapter/datasets.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from enum import Enum, unique
+from pathlib import Path
+
+from pydantic import BaseModel, ByteSize
+
+
+class DatasetMetaData(BaseModel):
+    id: str
+    display_name: str
+
+
+@unique
+class DataType(str, Enum):
+    FILE = "FILE"
+    FOLDER = "FOLDER"
+
+
+class PackageMetaData(BaseModel):
```

Comment:
TIP: distinguish which schemas are incoming or outgoing. I define base classes  `InputSchema` , `OutputSchema`

Reply:
these are internals. I don't want camelcasing here. and also don't want to go through the full refactoring if possible.

Reply:
these are items inside a paged Get call. they can also be a single Get call. how are they supposed to be named?
XXItem? XXGet?